torch/onnx/symbolic_opset9.py

```diff
@@ -1910,3 +1910,11 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
     return g.op("Multinomial", log_input,
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
+
+
+@parse_args('v')
+def gelu(g, self):
+    sqrt = g.op('Sqrt', g.op('Constant', value_t=torch.tensor(2, dtype=torch.float)))
+    erf = g.op('Erf', g.op('Div', self, sqrt))
```

Comment:
What's the advantage here?

Reply:
In general is to re-use the symbolic functions in case there are special handlings. For instance in the `Div` symbolic function there is code that does implicit casting for scalars. Calling directly `g.op(...)` misses that part.